src/components/inputs/ship-import/ShipImport.svelte

```diff
@@ -1,9 +1,9 @@
 <script lang="ts">
-    import { EdsyParser } from '../../../typescript/parsers/EdsyParser';
-    import { CoriolisParser } from '../../../typescript/parsers/CoriolisParser';
-    import { Parser } from '../../../typescript/parsers/Parser';
-    import { selectedDistributor, selectedWeapons } from '../../../typescript/store';
-    import { InaraParser } from '../../../typescript/parsers/InaraParser';
+    import { EdsyParser } from '../typescript/parsers/EdsyParser';
+    import { CoriolisParser } from '../typescript/parsers/CoriolisParser';
+    import { Parser } from '../typescript/parsers/Parser';
```

Comment:
This relates to several different places: We can drop the `typescript` directory across the board. Most of the time it only contains a sub directory anyway. In general though I don't think having a typescript direct is necessary anyway